typescript/simple-mercari-web/src/components/Listing/Listing.tsx

```diff
@@ -23,44 +27,87 @@ export const Listing: React.FC<Prop> = (props) => {
 
   const onValueChange = (event: React.ChangeEvent<HTMLInputElement>) => {
     setValues({
-      ...values, [event.target.name]: event.target.value,
-    })
+      ...values,
+      [event.target.name]: event.target.value,
+    });
   };
   const onFileChange = (event: React.ChangeEvent<HTMLInputElement>) => {
     setValues({
-      ...values, [event.target.name]: event.target.files![0],
-    })
+      ...values,
+      [event.target.name]: event.target.files![0],
+    });
   };
   const onSubmit = (event: React.FormEvent<HTMLFormElement>) => {
```

Comment:
同じく`async`関数にしてあげましょう
```suggestion
  const onSubmit = async(event: React.FormEvent<HTMLFormElement>) => {
```